OrbitService/main.cpp

```diff
@@ -49,4 +49,6 @@ int main(int argc, char** argv) {
   OrbitService service{asio_port};
   exit_requested = false;
   service.Run(&exit_requested);
+
+  grpc_server->Shutdown();
```

Comment:
This is why it was a draft. Is this needed/required? Since I added handling Ctrl-C...

Reply:
Yes, and Wait()